test-utils/fixtures/BaseFixtures.scala

```diff
@@ -53,6 +55,77 @@ trait BaseFixtures {
   val testEori = ConsigneeExportVat(ConsigneeExportVatType.YesEoriNumber, None, Some("1234"))
   val testVat = ConsigneeExportVat(ConsigneeExportVatType.YesVatNumber, Some("1234"), None)
 
+  val testExciseProductCodeW200: ExciseProductCode =
+    ExciseProductCode(
+      "W200",
+      "Still wine and still fermented beverages other than wine and beer",
+      "W",
+      "Wine and fermented beverages other than wine and beer"
+    )
+  val testExciseProductCodeW300: ExciseProductCode =
+    ExciseProductCode(
+      "W300",
+      "Still wine and still fermented beverages other than wine and beer",
```

Comment:
Not that it makes much difference:
```suggestion
      "Sparkling wine and sparkling fermented beverages other than wine and beer",
```